evals/agent_evals/run_test_case.py

```diff
@@ -0,0 +1,41 @@
+import os
+from typing import Any, Dict, List, Optional, Tuple
+from openai import OpenAI
+
+
+def get_open_ai_completion_function_params_for_agent(user_task: str, model: str, system_prompt: str, conversation_history: Optional[List] = []) -> Dict[str, Any]:
+    if not conversation_history:
+        conversation_history = [
+            {"role": "system", "content": system_prompt},
+            {"role": "user", "content": user_task}
+        ]
+    else:
+        conversation_history.append({"role": "user", "content": user_task})
+
+    completion_function_params = {
+        "model": model,
+        "messages": conversation_history,
+    }
+    if model == "gpt-4o-mini":
+        completion_function_params["temperature"] = 0.0
+
+    return completion_function_params
+
+
+def get_code_block_from_message(message: str) -> str:
+    # If ```python is not part of the message, then we assume that the
+    # entire message is the code block
+    if "```python" not in message:
+        return message
+
+    return message.split('```python\n')[1].split('\n```')[0]
+
+def get_openai_code_and_conversation_history(user_task: str, model: str, system_prompt: Optional[str] = None, conversation_history: Optional[List] = []):
```

Comment:
It looks like we are never returning the `conversation_history` from this function, so maybe this function should just be called `get_openai_code`